src/components/Sidebar.tsx

```diff
@@ -1,58 +1,118 @@
 import Link from "next/link";
 import Image from "next/legacy/image";
 import { signOut } from "next-auth/react";
+import {
+  faHouse,
+  faUsers,
+  faUserGroup,
+  faUserPlus,
+  faCircleUser,
+  faCity,
+  faArrowRightFromBracket,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
-const SidebarItem = ({ label } : { label: string }) => {
+interface Button {
+  name: string;
+  color: string;
+  icon: IconDefinition;
+}
+
+const userLinks: Button[] = [
+  {
+    name: "Home",
+    color: "#000000",
+    icon: faHouse,
+  },
+  {
+    name: "Students",
+    color: "#000000",
+    icon: faUsers,
+  },
+  {
+    name: "Seniors",
+    color: "#000000",
+    icon: faUserGroup,
+  },
+  {
+    name: "Pending",
+    color: "#000000",
+    icon: faUserPlus,
+  },
+  {
+    name: "Profile",
+    color: "#000000",
+    icon: faCircleUser,
+  },
+];
+
+const SidebarItem = ({
+  label,
+  iconName,
+}: {
+  label: string;
+  iconName: IconDefinition;
+}) => {
   return (
-    <div className="w-full px-4 p-2 text-md text-left font-serif duration-150 hover:bg-darker-tan">
-      <div className="duration-150 hover:translate-x-1">{label}</div>
+    <div className="text-md flex w-full space-x-4 p-2 px-10 text-left font-serif duration-150 hover:translate-x-1">
+      <div className="w-1/6 duration-150 hover:translate-x-1">
+        <FontAwesomeIcon icon={iconName} />
+      </div>
+      <div className="">{label}</div>
     </div>
   );
 };
 
-const Sidebar = ({ displayName } : { displayName: string }) => {
+const Sidebar = (/*{ displayName }: { displayName: string }*/) => {
   return (
-    <div className="flex flex-col w-[160px] h-screen top-0 py-12 justify-start gap-20 \
-                    bg-med-tan border-r border-darker-tan">
-          <div className="flex flex-col p-4 place-items-center">
-            <Link href="/">
-              <h1 className="font-serif text-center text-2xl duration-150 hover:-translate-y-0.5 z-10">
-                The Legacy Project
-              </h1>
-            </Link>
-
-            <span className="absolute h-[60px] aspect-square left-3 top-[48px] rotate-45 opacity-60">
-              <Image 
-                src="/landing/pink-flower.png"
-                layout="fill"
-                objectFit="cover"
-                // TODO(nickbar01234) - Write a more descriptive alt
-                alt="Pink flower"
-              />
-            </span>
-
-            <span className="absolute h-[60px] aspect-square top-[70px] left-[88px] opacity-40">
-              <Image 
-                src="/landing/yellow-flower.png"
-                layout="fill"
-                objectFit="cover"
-                // TODO(nickbar01234) - Write a more descriptive alt
-                alt="Yellow flower"
-              />
-            </span>
+    <div
+      className="\ top-0 flex h-screen w-[210px] flex-col justify-start border-r
```

Comment:
I think this should be `h-full` instead of `h-screen` so that it takes up the full screen. We probably don't want a hard code width like `w-[210px]`. The caller of `Sidebar` will probably set up a grid structure to fit in the navigation bar, so we can have it so that it fits the entire available width.

Additionally, this item can be `flex flex-col justify-between`. Notice in the Figma how the items are on separate side. This would require you to change how you layout the divs though.